src/components/RepoFileTree.js

```diff
@@ -23,7 +20,7 @@ const RepoFileTree = () => {
           <Cell
             css={{
               display: 'inline-grid',
-              gridTemplateColumns: '18px 18px 1fr',
+              gridTemplateColumns: '22px 22px 1fr',
```

Comment:
Not sure if this actually did anything... I did this in the middle of the night and didn't actually check 😅.